src/release/issue-triaging.md

```diff
@@ -54,6 +54,16 @@ To see a list of all labels, check out the "labels" page next to the search bar
 [IRLO]: https://internals.rust-lang.org/
 [Discord]: https://discord.gg/rust-lang
 
+### ICE Triage
+For [Issues that have both I-ICE and needs-triage](https://github.com/rust-lang/rust/issues?q=is%3Aissue%20state%3Aopen%20label%3AI-ICE%20label%3Aneeds-triage)
+* If it an older (like latest stable) version of rust, ask for or check the latest nightly (potentially check for duplicates)
+* If it does not have a reproduction, comment asking for one and add S-needs-repro. if there isn't one for around a month it should generally be closed.
+* If the reproduction is not minimal, add `E-needs-mcve` or create a Minimal Complete and Verifyable Example yourself.
+* Add `A-*` labels based on the code that causes the issue (check backtraces!),
```

Comment:
Suggestion: also check if:

- ICE requires debug-assertions, add `requires-debug-assertions` as suitable.
- ICE requires incomplete features, add `requires-incomplete-features` as suitable.
- ICE requires unstable features, add `F-*` name as suitable.
- ICE requires internal features, close as won't-fix (in accordance with [MCP 620](https://github.com/rust-lang/compiler-team/issues/620)), but double check if it ICEs *without* needing internal features.

Reply:
i already say to add F- and requires- labels?

i'll be honest, i'm finding this review a bit scattered and hard to follow, although i guess that's because i'm on the "Conversations" view.

Reply:
I'll post the feedback as one collectibe comment because the ui is extremely confusing for both you and me